package.json

```diff
@@ -23,6 +23,14 @@
   "homepage": "https://github.com/edopore/workshop-api-testing-js#readme",
   "devDependencies": {
     "chai": "^4.3.6",
+<<<<<<< HEAD
```

Comment:
This means that there was a conflict that was not solved.
Please solve the conflict or remove this symbols